rdg_datasets/arxiv_100_v1/migrate.py

```diff
@@ -0,0 +1,13 @@
+import os
+import pathlib
+
+from libuprev import rdg_migrate
+from libuprev.uprev_config import Config
+
+# vars unique to this rdg
+local_path = pathlib.Path(os.path.dirname(os.path.realpath(__file__)))
+input_storage_format_version = 1
```

Comment:
this should be 6 since the first available storage format version is 6

Reply:
done